common/src/main/java/gov/cms/ab2d/common/model/CoverageSummary.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.ab2d.common.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.List;
+
+/**
+ * Summary of coverage membership for a specific contract over a set of date ranges.
+ *
+ * This does not necessarily reflect a search over the entire lifetime of a contract only
+ * whatever date ranges are included.
+ */
+@Getter
+@ToString
+@AllArgsConstructor
+public class CoverageSummary {
+
+    private String beneficiaryId;
+    private Contract contract;
+    private List<DateRange> dateRanges;
```

Comment:
So, there can be gaps in the ranges, correct?

Reply:
Yes there can be I am copying the existing DateRange class in the worker module and upgrading it to java.time because java.time is much simpler to work with. This is actually what I think the most controversial change will be. See the class here: filter/src/main/java/gov/cms/ab2d/filter/FilterOutByDate.java